routers/api/v1/repo/release_tags.go

```diff
@@ -59,3 +61,54 @@ func GetReleaseTag(ctx *context.APIContext) {
 	}
 	ctx.JSON(http.StatusOK, convert.ToRelease(release))
 }
+
+// DeleteReleaseTag delete a tag from a repository
+func DeleteReleaseTag(ctx *context.APIContext) {
+	// swagger:operation DELETE /repos/{owner}/{repo}/releases/tags/{tag} repository repoDeleteReleaseTag
```

Comment:
the `/repos/{owner}/{repo}/tags` endpoint already exist, is there a reason th create a new one?

Reply:
Yeah, I also wasn't entirely sure about that, but this one seemed to already have a similar feel to it. 

I don't have a very strong opinion for either, let me know if the other would be preferred. It's a little hard to differentiate because tags are tied to releases like they are.